### ISSUE TITLE
Crash if permission are not granted

### ISSUE THREAD

**Issue by alex-ioma** [Anze/KeyCluCask#51]
After updating to 0.19 from 0.18, the setting page always crashes and cannot be open if the Accessibility and permissions are not granted. With said permission, if you try starting the setting page the app crashes and quits.

**Comment by alex-ioma** [comment]
Hi @Anze there is no CrashReporter file for KeyClu but I'll send the rest via email. 

Thanks.

**Comment by Anze** [comment]
Hey [alex-ioma](https://github.com/alex-ioma),
Thanks again for the collaboration. I'll keep the provided suggestions in mind. 
I've identified an issue, and it will be addressed in the upcoming release.

**Comment by alex-ioma** [comment]
You are welcome! I'll let you close this issue once it is resolved.

**Comment by Anze** [comment]
Hey [alex-ioma](https://github.com/alex-ioma),
New released is available now. Please let me know if you encounter any issues.

**Comment by alex-ioma** [comment]
Thank you. New version works well, although I'd still like to b able to
hide KeyClu icon at startup. That is too invasive imho.
